src/main/java/org/HdrHistogram/AbstractHistogram.java

```diff
@@ -2181,7 +2181,7 @@ int getBucketsNeededToCoverValue(final long value) {
      * value if we consider the sub-bucket length to be halved.
      */
     int getLengthForNumberOfBuckets(final int numberOfBuckets) {
-        final int lengthNeeded = (numberOfBuckets + 1) * (subBucketCount / 2);
+        final int lengthNeeded = (numberOfBuckets + 1) * (subBucketHalfCount);
```

Comment:
Was this an oversight, or was it an intentional choice to use the "cold" field (`subBucketCount`) instead of the "hot" `subBucketHalfCount` perhaps?